_pages/en/strategy-communications.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Communications Strategy
+ref: strategy-communications
+lang: en
+status: posted
+sections: Work In Progress
+permalink: /strategy-communications.html
+---
+
+## Communications Strategy
+
```

Comment:
Add Heading "Stakeholder identification"

1. IITB Senior Executives (DG and ADM)
2. IITB Executives (Directors and Executive Directors)
3. IITB Management (Managers and Team Leads)
4. IITB Workforce 
5. Non-IITB ESDC executives (e.g. "The Business")
6. General ESDC or other Government department personnel

Reply:
For each Stakeholders, we need to identify the key messages they will need to hear from our Team
E.g. IITB Senior Executives: General Direction, Endorsements, and progress updates towards that direction via Identified Metrics
E.g. IITB Management: Staffing-related and Management-related strategies (e.g. continuous learning objectives)
e.g. IITB Workforce: daily actions they can start doing today, tools at their disposal, forums at their disposal


Hey "key messages" is essentially an objective we want. We can then link to the "Communications Methods" at our disposal (next section) that can be used